src/library/Uno.Material/Styles/Controls/NavigationView/NavigationView_MUX.xaml

```diff
@@ -1009,17 +1009,19 @@
 										<Setter Target="RootGrid.Background" Value="{ThemeResource MaterialMUXTopNavigationViewItemBackgroundPointerOver}" />
 										<Setter Target="PointerRectangle.Fill" Value="{ThemeResource MaterialMUXNavigationViewItemBackgroundPointerOver}" />
 										<Setter Target="Icon.Foreground" Value="{ThemeResource MaterialMUXTopNavigationViewItemForegroundPointerOver}" />
-										<contract4Present:Setter Target="RootGrid.(media:RevealBrush.State)"
-																 Value="PointerOver" />
+										<!--Removing RevealBrush usage for now as it is not present in WinUI 3-->
```

Comment:
Can we maybe associate an issue to follow up on this later and precise the link with the comments ?

Reply:
I'm not sure what kind of issue to create, it's because RevealBrush was removed in WinUI 3 so it's not really a bug on our end. Unless, @jeromelaban do we know if they are planning to bring RevealBrush back later on?

Reply:
@kazo0 they may, but it's not clear yet so for now let's keep it commented out.